priv/templates/lvn.swiftui.gen/core_components.ex

```diff
@@ -251,18 +251,17 @@ defmodule <%= inspect context.web_module %>.CoreComponents.<%= inspect context.m
   """
   @doc type: :component
 
-  attr :class, :string, default: nil
```

Comment:
I think we'll want to retail the `class` attr on these components. We just need to add `style`

Reply:
This component just adds content to the navigation bar (for the title, subtitle, or toolbar). So adding additional classes shouldn't have any effect.

Reply:
In other cases, `class` should be caught by `:rest`.

Reply:
yes, good point